### PR TITLE
Update master with 6.4.1 Release

### PR DESCRIPTION
**Pull request by danfinlay**

### CODE REVIEW

File: ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js [hunk 1]
```diff
@@ -543,7 +543,7 @@ export default class ConfirmTransactionBase extends Component {
         toName={toName}
         toAddress={toAddress}
         showEdit={onEdit && !isTxReprice}
-        action={actionKey && this.context.t(actionKey) || getMethodName(name) || this.context.t('contractInteraction')}
+        action={this.context.t(actionKey) || getMethodName(name) || this.context.t('contractInteraction')}
```
Comment (tmashuang) [inline ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js]:
```suggestion
        action={actionKey || getMethodName(name) || this.context.t('contractInteraction')}
```

Comment (jennypollack) [reply ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js]: does this remove the [undefined] ?

Comment (tmashuang) [reply ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js]: It does, but will need to make a separate PR that targets `develop`

Comment (tmashuang) [reply ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js]: It will fallback to `contractInteraction`.

**Review by tmashuang — APPROVED** [review]

**Pull request merged**